datadog/tests/cassettes/TestAccDatadogAuthNMapping_CreateUpdate.yaml

```diff
@@ -0,0 +1,951 @@
+---
```

Comment:
curiosity: what is this and the `.freeze` file for ?

Reply:
These 2 files were generated by recording server responses while running acceptance test cases `RECORD=true make testall`. 
Going forward, for acceptance tests the idea is to use these server responses as mocks instead of actually CRUDing a resource from the server. 
The .yaml is the server response and the .freeze file is the datetime of when the response was captured.